README.md

```diff
@@ -116,12 +116,29 @@ The list of available actions can be seen via `medic-conf --help`.
 
 * upload of custom translations to the server
 
-## create-users **[ALPHA]**
+## create-users
```

Comment:
Should the create-users section come after the csv-to-docs section in the README.md? Since it is required that you undersatnd csv-to-docs before understanding create-users?